target_chains/ethereum/contracts/contracts/aave/PythPriceOracleGetter.sol

```diff
@@ -47,19 +57,34 @@ contract PythPriceOracleGetter is PythAssetRegistry, IPriceOracleGetter {
             PythAssetRegistry.validTimePeriodSeconds()
         );
 
-        // price.price is always positive and therefore this cast is safe.
+        // Aave is not using any price feeds < 0 for now.
+        if (price.price <= 0) {
+            revert InvalidNonPositivePrice();
+        }
         uint256 normalizedPrice = uint64(price.price);
         // price.expo is bound between [-12, 12]
-        int32 normalizerExpo = price.expo + int8(BASE_EXPONENTS);
+        int32 normalizerExpo = price.expo + int8(BASE_NUM_DECIMALS);
```

Comment:
the comment ^ isn't necessary because you have the check below right? I think a better comment would be on the check to say this prevents overflow in normalizedPrice.